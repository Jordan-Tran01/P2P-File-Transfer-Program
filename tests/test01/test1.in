ADDPACKAGE part2_tests/test01/test1.bpkg
PACKAGES
QUIT