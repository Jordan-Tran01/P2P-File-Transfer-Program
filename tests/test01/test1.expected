1. 3cf007c14ded16ab85d168fcf9d9b20e, btide_test1/pkg1.data : INCOMPLETE
Identifier provided does not match managed packages
1. 3cf007c14ded16ab85d168fcf9d9b20e, btide_test1/pkg1.data : INCOMPLETE
Package has been removed
No packages managed
